cola/themes.py

```diff
@@ -551,8 +570,26 @@ def get_all_themes():
         Theme(
             'flat-dark-green', N_('Flat dark green'), True, EStylesheet.FLAT, '#42a65c'
         ),
+        Theme(
+            'system', N_('System'), None, EStylesheet.SYSTEM, None
+        ),
```

Comment:
Note: `'system'` looks like it's identical to `'default'` -- what's with the `False` vs. `None` difference in the 3rd parameter?

Do we need `'system'` in light of `'default'`?

Reply:
removed system. As explained above, it was a problem in Ubuntu 20.04 but not 22.04